op-node/rollup/derive/test/random.go

```diff
@@ -0,0 +1,23 @@
+package test
+
+import (
+	"math/rand"
+
+	"github.com/ethereum-optimism/optimism/op-node/eth"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-node/testutils"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/trie"
+)
+
+// RandomL2Block returns a random block whose first transaction is a random
+// Deposit transaction.
```

Comment:
```suggestion
// L1 Info Deposit transaction.
```

Reply:
Added. Also randomized the `regolith` bool.